apps/functions/applications-migration/common/migrators/s51-advice-migration.js

```diff
@@ -42,21 +42,22 @@ export const migrateS51Advice = async (logger, caseReferences) => {
  * @param {import('@azure/functions').Logger} log
  * @param {string} caseReference
  */
-export async function migrateS51AdviceForCase(log, caseReference) {
+export async function migrateS51AdviceForCase(log, caseReference, synapseQuery = query) {
 	try {
 		log.info(`reading S51 Advice with caseReference ${caseReference}`);
 
-		const [s51AdviceRows, count] = await SynapseDB.query(query, {
+		const [s51AdviceRows, count] = await SynapseDB.query(synapseQuery, {
 			replacements: [caseReference]
 		});
 
 		const s51AdviceEntities = s51AdviceRows.map((row) => {
-			removeValues(row, [null, 'None']);
-
+			if (caseReference === ODW_GENERAL_CASE_REF) {
+				mapGeneralToBoCaseRef(row);
+			}
```

Comment:
Is the `mapGeneralToBoCaseRef` used elsewhere? If not I think it would be clearer to just inline it here.

```suggestion
			if (caseReference === ODW_GENERAL_CASE_REF) {
				row.adviceReference = row.adviceReference.replace(ODW_GENERAL_CASE_REF, BO_GENERAL_CASE_REF);
				row.caseReference = BO_GENERAL_CASE_REF;
			}
```

Reply:
done!